src/app/main.c

```diff
@@ -121,7 +133,20 @@ option_context_new_with_commands (RpmOstreeCommandInvocation *invocation,
     {
       gboolean hidden = (command->flags & RPM_OSTREE_BUILTIN_FLAG_HIDDEN) > 0;
       if (!hidden)
-        g_string_append_printf (summary, "\n  %s", command->name);
+        {
+          g_string_append_printf (summary, "\n  %s", command->name);
+          if (command->description != NULL)
+            {
+              /* add padding for description alignment */
+              guint max_command_len = 15;
+              guint pad = max_command_len - strlen (command->name);
+
+              for (guint padding_num = 0; padding_num < pad + 2; padding_num++)
+                g_string_append (summary, " ");
```

Comment:
Instead of all this, you should be able to use `printf`'s native padding. Check out the "Field width" section in `printf(3)`.